components/Form/index.js

```diff
@@ -113,6 +130,52 @@ export default function Form() {
             </div>
           );
         })}
+        {additionalItems.map((item, index) => {
+          return (
+            <div key={item.id}>
```

Comment:
Since this is a form, there should be a `form` element as a wrapper; is this in another component? 

You can use a `<React.Fragment key={item.id}>` instead of a `div`; by doing so, there won't be an unnecessary `div`.

Reply:
i want to style the additionalItems later, so i will need it.

Reply:
If you want to style it later, you can change it to a `div` later.